### PR TITLE
various linting fixes

### PR DESCRIPTION
**Pull request by Richard-Burd**

### CODE REVIEW

File: api/src/directives/requireAuth/requireAuth.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const schema = gql`
 type RequireAuthValidate = ValidatorDirectiveFunc<{ roles?: string[] }>
```
Comment (stormcloud266) [inline api/src/directives/requireAuth/requireAuth.ts]: I believe changing the roles typing here to `SiteRole[]` will let you undo the changes to `src/lib/auth`

**Review by stormcloud266 — CHANGES_REQUESTED** [review]

File: api/src/lib/keyp/oAuth/helpers.ts [hunk 2]
```diff
@@ -1,7 +1,10 @@
-export const getExpiration = (expiresIn) =>
+export const getExpiration = (expiresIn: number): Date =>
   new Date(new Date(Date.now() + expiresIn * 1000))
 
-export const encodeBody = (body) =>
+export const encodeBody = (body: Record<string, unknown>): string =>
```
Comment (illepic) [inline api/src/lib/keyp/oAuth/helpers.ts]: `(body: Record<string, string>): string`

File: api/src/lib/verifySignature.ts [hunk 3]
```diff
@@ -12,7 +16,7 @@ export const getSignature = async (
 ) => {
   const contractAddress = contractAddressLookup[chainId]
```
Comment (illepic) [inline api/src/lib/verifySignature.ts:19]:
```ts
if (!wallet) throw new Error('Wallet failed to initialize.');
```

place before 

```ts
const contractAddress = contractAddressLookup[chainId]
```

File: api/src/services/submissions/submissions.scenarios.ts
Comment (illepic) [inline api/src/services/submissions/submissions.scenarios.ts:1]: Revert this file, ignore all scenarios.

**Review by illepic — CHANGES_REQUESTED** [review]

**Review by illepic — APPROVED** [review]

**Review by stormcloud266 — APPROVED** [review]

**Pull request merged**